Sources/ClientRuntime/Config/DefaultHttpClientConfiguration.swift

```diff
@@ -8,6 +8,7 @@
 import protocol SmithyHTTPAPI.HTTPClient
 import protocol SmithyHTTPAuthAPI.AuthScheme
 import protocol SmithyHTTPAuthAPI.AuthSchemeResolver
+import protocol SmithyIdentity.AWSCredentialIdentityResolver
```

Comment:
In this file, several properties were added to `DefaultHttpClientConfiguration` to permit accessing them in reusable code.

No code change is required in the conforming client config objects, because all client configs already have these properties.

As with `DefaultClientConfiguration` above, this is technically a breaking change, but I don't expect that any customer is conforming their own types to this protocol.